android/java/org/chromium/chrome/browser/crypto_wallet/model/WalletListItemModel.java

```diff
@@ -143,7 +144,8 @@ public int getIcon() {
     }
 
     public String getTitle() {
-        return mTitle;
+        // ERC721 has format [Title #ID]
+        return mTitle + (isErc721() ? "" : "#" + Utils.hexToIntString(mId));
```

Comment:
is that correct? as I see we do `[Title #ID]` for non ERC721 now. Shouldn't it be an opposite?

Reply:
The if here is actually working correctly with the conditions in `isErc721`, just the function being misnamed. But for clarity I'll flip the conditions.